python/tests/test_module.py

```diff
@@ -19,6 +19,17 @@ def test_module(self):
 
         self.assertEqual(set(result),set(["module2","module1"]))
 
+    def test_generator(self):
+        result = []
+
+        s = uhdm.Serializer()
+
+        data = uhdm.buildTestDesign(s)
```

Comment:
If you have an better idea that to use a build-in fake design to non-regress because it appears to be not very representative of data that can be present in uhdm...

Reply:
> My plan is to rewrite the uhdm-hier and uhdm-dump utilities in python to demo how to use them

That would be excellent !